cypress/component/SignIn/sign_in_button.spec.js

```diff
@@ -35,24 +36,35 @@ const userStatus = {
   'tosAccepted': true
 };
 
+const consentStatus = {
+  systems: {
+    sam: {
+      details: {
+        ok: true
+      }
+    }
+  }
+};
+
 const notAcceptedUserStatus = Object.assign({}, userStatus, {'tosAccepted': false});
 
 describe('Sign In: Component Loads', function () {
 
-  // Intercept configuration calls
   beforeEach(() => {
     cy.initApplicationConfig();
+    cy.stub(ServiceStatus, 'getConsentStatus').resolves(consentStatus);
```

Comment:
For some reason if I add this to the tests instead, the tests fail. This is very mysterious `cypress` behavior to me.